acestep/ui/gradio/events/generation/metadata_loading.py

```
@@ -141,17 +143,212 @@ def load_metadata(file_obj, llm_handler=None):
+        return _save_project_impl(
+            task_type, captions, lyrics, vocal_language, bpm, key_scale, time_signature,
+            audio_duration, batch_size_input, inference_steps, guidance_scale, seed,
+            random_seed_checkbox, use_adg, cfg_interval_start, cfg_interval_end, shift,
+            infer_method, custom_timesteps, audio_format, mp3_bitrate, mp3_sample_rate,
+            lm_temperature, lm_cfg_scale, lm_top_k, lm_top_p, lm_negative_prompt,
+            use_cot_metas, use_cot_caption, use_cot_language, audio_cover_strength,
+            cover_noise_strength, think_checkbox, text2music_audio_code_string,
+            repainting_start, repainting_end, track_name, complete_track_classes,
+            instrumental_checkbox, song_name,
+        )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)